src/main/java/com/github/onsdigital/JWTVerifierImpl.java

```diff
@@ -38,20 +39,20 @@ public class JWTVerifierImpl implements JWTVerifier {
      * @throws IllegalArgumentException if the public signing keys provided are invalid
      */
     public JWTVerifierImpl(Map<String, String> signingKeys) {
-        final SigningKeyResolver signingKeyResolver = new SigningKeyResolverImpl(signingKeys);
+        final Locator<Key> keyLocator = new KeyLocatorImpl(signingKeys);
```

Comment:
Looks good. I only would rename KeyLocaltor to SigningKeyLocator to be specific here. Could be a key to a hash but is not. So we give the reader a bit more context.